clash/force_of_will.tex

```diff
@@ -0,0 +1,101 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{Force of Will}{\images/berserk.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} Shakajiub
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1237891632448278618}{Archon Studios Discord}
+
+\textit{Wild Dragons are terrorizing the countryside. Farmers are unable to grow their crops in peace, caravans are scared to move their goods. You must strike at their core and defeat the beasts. If by nothing else, then by sheer force of will.}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+This scenario is played over 12 rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2 or 3
+
+\textbf{Starting Resources:}\par
+\resources{15}{2}{1}
+
+\textbf{Starting Income:}\par
+\resources{10}{2}{1}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item 2 × Few \svgunit{bronze} units with the \textit{highest} Recruitment cost
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling
+
+\textbf{Map tile Pool:} Each player takes 1 random Far (II--III) Map tile
+
+\textbf{Additional Bonus:} None
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map tiles and arrange them as shown in the scenario map layout:
+
+\textbf{For a 2-player scenario:}
+\begin{itemize}
+  \item 2 × Starting (I) Map tile
+  \item 2 × Near (IV--V) Map tile
+  \item 4 × Far (II--III) Map tile
+  \item 1 × Center (VI--VII) Map tile, which must contain the Dragon Utopia field
+\end{itemize}
+
+\textbf{For a 4-player scenario:}
+\begin{itemize}
+  \item 3 × Starting (I) Map tile
+  \item 3 × Near (IV--V) Map tile
+  \item 6 × Far (II--III) Map tile
+  \item 1 × Center (VI--VII) Map tile, which must contain the Dragon Utopia field
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+To win the scenario, a Hero must Flag the Dragon Utopia field.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+At the end of the \nth{12} Round, if there is no winner, all players lose the scenario.
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{\nth{4} and \nth{8} Rounds:}
+\begin{itemize}
+  \item Remove all Black cubes from every Water Wheel and Windmill on the map.
+\end{itemize}
+
+\textbf{(2-P Scenario Only) \nth{5} and \nth{9} Rounds:}
+\begin{itemize}
+  \item The second player gains 1 \svg{movement}.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+
+During this scenario:
+
+\begin{itemize}
+  \item When a player Visits an Obelisk, they roll 2 \svg{treasure} and choose one to resolve.
+  \item A player may not use the Diplomacy card to skip Combat on the Dragon Utopia field.
+  \item When a Hero moves to the Dragon Utopia field, they must fight, depending on chosen difficulty, the following number of Neutral units:
+  \begin{itemize}[leftmargin=15pt]
+    \item \textbf{Easy:} 1 × \svg{azure}, 2 × \svg{silver}, 1 × \svg{bronze}
```

Comment:
```suggestion
    \item \textbf{Easy:} 1 × \svgunit{azure}, 2 × \svgunit{silver}, 1 × \svgunit{bronze}
```
Please do the same for other unit glyphs.

Reply:
Why was there a change from `svg` to `svgunit`? Why `unit`?

Reply:
Yes, stars have a different drop than the rest of the glyphs. The stars didn't look so good when they were so low.

Reply:
I see, so it's only meant to be for stars/units/tiers. Maybe it needs some comment for usage (not necessarily in this PR). It wasn't clear to me in first glance